README.adoc

```diff
@@ -68,3 +68,27 @@ It is in link:.gitignore[.gitignore] so that you can keep your test scenarios un
 == Contributing
 
 Please submit issues and pull requests to the project. You can discuss and ask questions on https://lists.openshift.redhat.com/openshiftmm/listinfo/dev[OpenShift mailing list].
+
+
+=== RVM for dependency management
+
+If you are planning on installing dependencies for verification Tests on your system, it is usually a good idea to use some kind of Version Manager/Package manager, like RVM.
+
+RVM is analogous to virtualenv in python and it stands for Ruby Version Manager.
+
+You can install RVM and read more abou it on https://rvm.io/
+
+Usually once RVM is installed, you would want to create/install a new `rvm` interpreter using:
+
+```rvm install "ruby-2.7.1" && rvm  --create use 2.7.1@verification-tests``` which will install Ruby 2.7.1 and create a new interpreter as well as a Gemset.
+When we install Ruby 2.7.1, it creates a local directory for your user with Ruby 2.7.1 under `/home/<user>/.rvm/rubies/ruby-2.7.1`.
```

Comment:
aka `$HOME` or `~/`?

Reply:
good suggestion. Didn't know if there could be other home dirs haha. Never heard of DYNIX. I will update this.

Reply:
Sounds like some UNIX variant. It is usual to see home dirs in non-standard locations where shared storage is being used.

Reply:
just `~/.rvm` would be sufficient.  For Mac user it's `/Users/pruan/.rvm`.  Anyway,  `~/.rvm` would work regardless.

Reply:
Updated this.